src/v1/bookSearch/RecentSearch.tsx

```diff
@@ -0,0 +1,41 @@
+import type { APIBookRecentSearchResponse } from '@/types/book';
+
+import Button from '@/v1/base/Button';
+
+type RecentSearchProps = {
+  recentSearches: APIBookRecentSearchResponse[] | undefined;
```

Comment:
p5;
Optional 타입으로 변경해도 좋을 것 같아요~
```suggestion
  recentSearches?: APIBookRecentSearchResponse[]
```

Reply:
[f3e1ab8](https://github.com/prgrms-web-devcourse/Team-Gaerval-Dadok-FE/pull/478/commits/f3e1ab8b674000fe84ad07dfabe7bb41eab33584) 반영하였습니다~!